src/compartment.c

```diff
@@ -53,7 +52,10 @@ entry_point_cmp(const void* val1, const void* val2)
  * a `struct Compartment`. At this point, we only read data.
  */
 struct Compartment*
-comp_from_elf(char* filename, struct ConfigEntryPoint* entry_points, size_t entry_point_count)
+comp_from_elf(char* filename,
+              char** entry_points, size_t entry_point_count,
+              char** intercepts, uintptr_t* intercept_addrs, size_t intercept_count,
+              uintptr_t new_comp_base)
```

Comment:
Should this be `void *new_comp_base`?

Reply:
Changed in cfe4bf91f55c6b75ad38d42cd7b660c9faa2d3d2.